test/e2e/util/util.go

```diff
@@ -79,8 +79,16 @@ func createCluster(t *testing.T, ctx context.Context, client crclient.Client, cl
 			t.Logf("Skipping cluster dump because no artifact dir was provided")
 		}
 		// TODO: Figure out why this is slow
-		//e2eutil.DumpGuestCluster(context.Background(), client, hostedCluster, globalOpts.ArtifactDir)
-		DestroyHostedCluster(t, ctx, hc, clusterMgr, artifactDir)
+		// e2eutil.DumpGuestCluster(context.Background(), client, hostedCluster, globalOpts.ArtifactDir)
+		destroyAttempt := 1
```

Comment:
What is the purpose of this variable? There isn't any retrying, it is just logged once, then incremented, then unused

Reply:
DestroyHostedCluster retries internally via PollImmediateUntil